src/autocomplete/index.ts

```diff
@@ -1,254 +1,60 @@
+import {AutocompleteParseResult} from './lib/types';
 import {clickhouseAutocompleteParser} from './parsers/clickhouse/clickhouseAutocompleteParser';
 import {genericAutocompleteParser} from './parsers/generic/genericAutocompleteParser';
 import {postgresqlAutocompleteParser} from './parsers/postgresql/postgresqlAutocompleteParser';
 
+export {
+    ErrorLocation as ParserSyntaxError,
+    AutocompleteParseResult as ParseResult,
+    ValuesSuggestion,
+    TablesSuggestion,
+    ColumnSuggestion,
+    AggregateFunctionsSuggestion,
+    ColumnAliasDetails,
+    DatabasesSuggestion,
+    FiltersSuggestion,
+    FunctionsSuggestion,
+} from './lib/types';
```

Comment:
Let's rename this file to `lib/autocomplete-parse-result.ts`? I don't like naming files too generally like `types` or `objects`, because `types` kinda implies that every type we have in the project there. But `lib/autocomplete-parse-result.ts` says that it's only types for the parse result

Reply:
We still have `lib/types.ts`. We need to dot it for `lib/types.ts`